src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -949,7 +949,8 @@ def update_connected_cluster_internal(client, resource_group_name, cluster_name,
 
 def update_connected_cluster(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="",
                              disable_proxy=False, kube_config=None, kube_context=None, auto_upgrade=None, tags=None,
-                             distribution=None, distribution_version=None, azure_hybrid_benefit=None, yes=False, container_log_path=None):
+                             distribution=None, distribution_version=None, azure_hybrid_benefit=None, yes=False, container_log_path=None,
+                             enable_oidc_issuer=False):
```

Comment:
For brownfield scenario, there could be some customers who have agent auto update disabled and might not have latest helm chart changes for oidcenabled flag. we should add this check and give warning to those customers to upgrade their agent version to latest if they want to enable this feature